playbulb-candle/playbulbCandle.js

```diff
@@ -114,6 +116,20 @@
         this._isEffectSet = true;
       });
     }
+    startBlowNotifications(listener) {
+      let characteristic = this._characteristics.get(CANDLE_BLOW_NOTIFICATIONS_UUID);
+      return characteristic.startNotifications()
+      .then(_ => {
+        characteristic.addEventListener('characteristicvaluechanged', listener);
+      });
+    }
+    stopBlowNotifications(listener) {
+      let characteristic = this._characteristics.get(CANDLE_BLOW_NOTIFICATIONS_UUID);
+      return characteristic.stopNotifications()
+      .then(_ => {
+        characteristic.removeEventListener('characteristicvaluechanged', listener);
```

Comment:
I'd have removed the listener before stopping notifications, to make as much synchronous as possible. Do you have a reason to prefer doing it after? The choice doesn't matter much, but we should try to set a considered example.